From a10bfe927a016f0f8dc1afe03261af5c4cd7b1bc Mon Sep 17 00:00:00 2001
From: Ofer Heifetz <oferh@marvell.com>
Date: Sun, 31 Dec 2017 15:17:37 +0200
Subject: [PATCH 2143/2241] Revert "crypto: inside-secure: update eip197d TRC
 RAM size"

pushed by mistake on HEAD without CI/gerrit

This reverts commit d34b9a04f1832a1e1d3becd898e042d57ea34c5a.
---
 drivers/crypto/inside-secure/safexcel.c | 10 ++--------
 drivers/crypto/inside-secure/safexcel.h |  6 ++----
 2 files changed, 4 insertions(+), 12 deletions(-)

diff --git a/drivers/crypto/inside-secure/safexcel.c b/drivers/crypto/inside-secure/safexcel.c
index db66633..6c206e5 100644
--- a/drivers/crypto/inside-secure/safexcel.c
+++ b/drivers/crypto/inside-secure/safexcel.c
@@ -81,10 +81,7 @@ static int eip197_trc_cache_init(struct device *dev,
 	/* Calculate the maximum possible record count that
 	 * the Record Cache Data RAM can contain
 	 */
-	if (priv->eip197_hw_ver == EIP197B)
-		rc_record_cnt = EIP197B_TRC_RAM_WC / rc_rec_wc;
-	else
-		rc_record_cnt = EIP197D_TRC_RAM_WC / rc_rec_wc;
+	rc_record_cnt = EIP197_TRC_RAM_WC / rc_rec_wc;
 
 	/* rc_record_cnt is calculated using the configured RC Data RAM size. */
 
@@ -93,10 +90,7 @@ static int eip197_trc_cache_init(struct device *dev,
 	 */
 
 	/* set the configured RC Admin RAM size */
-	if (priv->eip197_hw_ver == EIP197B)
-		rc_admn_ram_wc = EIP197B_TRC_ADMIN_RAM_WC;
-	else
-		rc_admn_ram_wc = EIP197D_TRC_ADMIN_RAM_WC;
+	rc_admn_ram_wc = EIP197_TRC_ADMIN_RAM_WC;
 
 	/* Calculate the maximum possible record count that
 	 * the RC Hash Table (in Record Cache Administration RAM) can contain
diff --git a/drivers/crypto/inside-secure/safexcel.h b/drivers/crypto/inside-secure/safexcel.h
index 4c66bb1..5dee660 100644
--- a/drivers/crypto/inside-secure/safexcel.h
+++ b/drivers/crypto/inside-secure/safexcel.h
@@ -65,10 +65,8 @@
 #define EIP197_RC_NULL					0x3FF
 #define EIP197_RC_HASH_TABLE_MASK			(GENMASK(29, 0))
 /* transformation */
-#define EIP197B_TRC_ADMIN_RAM_WC			320
-#define EIP197B_TRC_RAM_WC				3840
-#define EIP197D_TRC_ADMIN_RAM_WC			640
-#define EIP197D_TRC_RAM_WC				7680
+#define EIP197_TRC_ADMIN_RAM_WC				320
+#define EIP197_TRC_RAM_WC				3840
 
 /* Record administration */
 #define EIP197_RC_HASH_COLLISION_PREV			20
-- 
2.7.4

